aws/aws-how-to/security/ubuntu-security-on-aws.rst

```diff
@@ -0,0 +1,47 @@
+Ubuntu Security on AWS
+######################
+
+Ubuntu security features
+************************
+
+Ubuntu on AWS provides all security features available to Ubuntu Server. For more detailed information about how Ubuntu offer security, refer to the following `Introduction to Security <https://documentation.ubuntu.com/server/explanation/intro-to/security/>`_.
+
+
+AWS security features
+*********************
+
+AWS offers comprehensive security and data protection in the cloud. `Security in Amazon EC2 <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security.html>`_ explains how user can benefit from AWS security features.
+
+Secure Boot and TPM
+~~~~~~~~~~~~~~~~~~~
+
+AWS Secure Boot is a feature providing secure boot process of Amazon EC2 so instance only boots trusted softwares. To create and configure secure boot image with Ubuntu AMI, check out :doc:`./use-secureboot-and-vtpm`.
+
+CIS hardened image
+~~~~~~~~~~~~~~~~~~
+
+CIS hardened images are built with configuration by the Center for Internet Security (CIS) for use on Amazon EC2.
+
+To create hardened image with AWS Ubuntu Pro image, follow instructions in `Comply with CIS or DISA STIG on Ubuntu 20.04 with Ubuntu Security Guide <https://ubuntu.com/tutorials/comply-with-cis-or-disa-stig-on-ubuntu#1-overview>`_.
+
+AMD SEV-SNP
+~~~~~~~~~~~
+
+AMD Secure Encrypted Virtualization-Secure Nested Paging (AMD SEV-SNP) provides strong memory integrity protection to instances that use AMD EPYC processors. Full details with how to launch AMD SEV-SNP instances are in :doc:`../instances/launch-and-attest-amd-sev-snp-instances`.
+
+Ubuntu Pro
+**********
+
+Ubuntu offers various type of AWS images to meet security requirements. Images can have enhanced security features with `Ubuntu Pro <https://ubuntu.com/aws/pro>`_.
+
+Images with Pro can offer:
+
+* Expanded Security Maintenance (ESM): Provide 10 years of ubuntu software package management.
+* Live kernel updates: Reduce downtime and unplanned reboots for kernel vulnerabilities.
+* FIPS: Configure system to meet strict security requirements creating highly regulated environments.
+
+:doc:`../instances/find-ubuntu-images` explains how to find Ubuntu Pro images for AWS.
+
+To learn more about Ubuntu Pro, check out `Ubuntu Pro for AWS <https://ubuntu.com/aws/pro>`_.
```

Comment:
Since this is the same link as the one specified at the beginning of this section, we can remove this line